controllers/helm.go

```diff
@@ -109,12 +110,28 @@ func detectChartDrift(combinedConfigs *k0sv1beta1.HelmExtensions, installedChart
 			if targetChart.Version != checkVersion {
 				chartDrift = true
 			}
+			targetValuesMap := map[string]interface{}{}
+			err := yaml.Unmarshal([]byte(targetChart.Values), &targetValuesMap)
+			if err != nil {
+				return nil, false, err
```

Comment:
Can we add a wrapper to this error with the chart name and the fact that it's a chart we want to apply?

Reply:
```suggestion
				return nil, false, fmt.Errorf("target chart %s values error: %w", targetChart.Name)
```